README.md

```diff
@@ -1,24 +1,585 @@
+            "errors": []
+        }
+    }
+}
+```
+#### Error Handling
+##### `INSERT-ERROR-CODE-HERE` INSERT-ERROR-EXPLAINATION-HERE
+##### Example Mutation
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### [Mutation New Encounter](#mutation-new-encounter)
+**`ENDPOINT BADGE HERE`**
+##### Example Mutation
+```graphql
+mutation {
+    createEncounter(input: {
+        copper: 1,
+        silver: 1,
+        gold: 1,
+        platinum: 1,
+        electrum: 1,
+        encounter_loots: [
+            {
+                item_name: "Longbow"
+            },
+            {
+                item_name: "Greataxe"
+            }
+        ],
+        encounter_monsters: [
+            {
+                monster_name: "Beholder"
+            }
+        ]
+    }) {
+        encounter {
+            id
+            copper
+            silver
+            gold
+            platinum
+            electrum
+            encounter_loots {
+                item_name
+            }
+            encounter_monsters {
+                monster_name
+            }
+        }
+        errors
+    }
+}
+```
+##### Example Response
+```json
+{
+    "data": {
+        "createEncounter": {
+            "encounter": {
+                "id": 55,
+                "copper": "1",
+                "silver": "1",
+                "gold": "1",
+                "platinum": "1",
+                "electrum": "1",
+                "encounter_loots": [
+                    {
+                        "item_name": "Longbow"
+                    },
+                    {
+                        "item_name": "Greataxe"
+                    }
+                ],
+                "encounter_monsters": [
+                    {
+                        "monster_name": "Beholder"
+                    }
+                ]
+            },
+            "errors": []
+        }
+    }
+}
+```
+#### Error Handling
+##### `INSERT-ERROR-CODE-HERE` INSERT-ERROR-EXPLAINATION-HERE
+##### Example Query
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- ROADMAP -->
+## Roadmap
+
+- [ ] Database Schema
+- [ ] Set Up Mock Server on Postman (browser version)
+    - [ ] JSON Contracts
+- [ ] Endpoint Task
+    - [ ] Sub-Tasks
+
+
+
+See the [open issues](https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues) for a full list of proposed features (and known issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTACT -->
+## Contact
+
+Organization: Tavern Keeper - INSERT-ORG-EMGAIL-HERE
+
+Organization Link: [https://github.com/Tavern-Keeper-2308](https://github.com/Tavern-Keeper-2308)
+
+Project Link: [https://github.com/Tavern-Keeper-2308/tavern_keeper_be](https://github.com/Tavern-Keeper-2308/tavern_keeper_be)
```

Comment:
Would it make sense to also include a link to the fe repo? Like, "Related Project Link" or something like that?